modules/tensor_mechanics/doc/content/documentation/systems/UserObjects/GeneralizedPlaneStrainUserObject.md

```diff
@@ -21,6 +21,12 @@ and the corresponding diagonal jacobian is
 	K_{zz} = \frac{\partial R_{zz}}{\partial \epsilon_{zz}} = \int_{A}{\frac{\partial \sigma_{zz}}{\partial \epsilon_{zz}}dA} = \int_{A}{C_{2222}dA}
 \end{equation}
 
+The reference residual value used by [GeneralizedPlaneStrainReferenceResidual](/systems/AuxScalarKernels/GeneralizedPlaneStrainReferenceResidual.md) is computed as
+
+\begin{equation}
+	v = \int_{A}{|\sigma_{zz}|dA}
+\end{equation}
```

Comment:
Define the variables in the equation please

Reply:
I'm going to try to make this clearer, but it seems pretty clear already.